web/apps/labelstudio/src/app/App.scss

```diff
@@ -9,6 +9,11 @@ body {
 
   margin: 0;
   font-family: Figtree, sans-serif;
+  display: flex;
```

Comment:
This possibly breaks the other pages, as flex would not be the default layout for body.